tests/flytekit/unit/clients/test_auth_helper.py

```diff
@@ -177,13 +175,6 @@ def test_upgrade_channel_to_proxy_auth():
     assert isinstance(out_ch._interceptor._authenticator, CommandAuthenticator)
 
 
-def test_load_cert():
-    cert_file = os.path.join(os.path.dirname(__file__), "testdata", "rootCACert.pem")
```

Comment:
Can you also remove the rootCACert.pem and rootCAKey.pem [files](https://github.com/flyteorg/flytekit/tree/master/tests/flytekit/unit/clients/testdata)?